### PR TITLE
Code quality improvements for SCIM Module

### PR DESCRIPTION
**Pull request by IndunilRathnayake**

### CODE REVIEW

File: components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/config/SCIMProviderDAO.java [hunk 1]
```diff
@@ -72,10 +73,9 @@
             log.error(errorMsg, e);
             throw new IdentitySCIMException(errorMsg, e);
         } catch (SQLException e) {
-            log.error("Error when executing the SQL : " + SQLQueries.GET_ALL_PROVIDERS_SQL);
-            log.error(e.getMessage(), e);
+            log.error("Error when executing the SQL : " + SQLQueries.GET_ALL_PROVIDERS_SQL, e);
             throw new IdentitySCIMException("Error when reading the SCIM Providers information from " +
```
Comment (hpmtissera) [inline components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/config/SCIMProviderDAO.java]: when we log the exception with stack trace as an error log, no need to throw with stack trace. fix like this

 log.error("Error when executing the SQL : " + SQLQueries.GET_PROVIDER_SQL, e);
             throw new IdentitySCIMException("Error when reading the SCIM Provider information from " +
                  "the persistence store.");

File: components/scim/org.wso2.carbon.identity.scim.ui/src/main/resources/web/scim/add-finish.jsp~ [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<!--
```
Comment (hpmtissera) [inline components/scim/org.wso2.carbon.identity.scim.ui/src/main/resources/web/scim/add-finish.jsp~]: remove this file,add-finish.jsp~ is a temp file

**Comment by IndunilRathnayake** [comment]
Fixed issues found in Code Review

**Comment by hpmtissera** [comment]
already merged

**Pull request closed without merging**